narayana-integration-bom/pom.xml

```diff
@@ -26,7 +26,7 @@
     <!-- DBCP connection pooling for Narayana -->
     <version.org.apache.tomcat.tomcat-dbcp>9.0.21</version.org.apache.tomcat.tomcat-dbcp>
 
-    <version.org.jboss.transaction.spi>7.6.0.Final</version.org.jboss.transaction.spi>
+    <version.org.jboss.transaction.spi>7.6.1.Final</version.org.jboss.transaction.spi>
     <version.jakarta.transaction-api>1.3.3</version.jakarta.transaction-api>
```

Comment:
_jboss-transaction-spi-jakarta_ pom has different versions for transaction-api and resource-api that are being downgraded here:

https://github.com/jbosstm/jboss-transaction-spi/blob/7.6.1.Final/jakarta/pom.xml#L30-L31

Reply:
FDB failed for unrelated errors. The downgraded versions for transaction-api and resource-api seem to integrate well with jboss-transaction-spi 7.6.1.Final version.